test/built-ins/Temporal/Duration/prototype/round/balances-up-to-weeks.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.prototype.round
+description: Balances up to weeks correctly when years and months are present.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const oneMonthOneDay = new Temporal.Duration(0, 1, 0, 1, 0, 0, 0, 0, 0, 0);
+const oneYearOneMonthOneDay = new Temporal.Duration(1, 1, 0, 1, 0, 0, 0, 0, 0, 0);
```

Comment:
I'd suggest something like
```js
const relativeTo = new Temporal.PlainDate(2024, 1, 1);
```
and using that in the options bags throughout.

Reply:
Also, I've been looking at the existing tests in `round/` and I think we might not have coverage of rounding a duration with only days up to weeks. So maybe add something like
```js
const severalWeeksInDays = new Temporal.Duration(0, 0, 0, 29, 0, 0, 0, 0, 0, 0);
```
(I might have missed it if we already had coverage of this, though! There are a lot of tests in this folder)

Reply:
Done and done.